packages/react-core/src/components/Menu/MenuListItem.tsx

```diff
@@ -129,6 +132,7 @@ export const MenuListItem: React.FunctionComponent<MenuListItemProps> = ({
             <span>{description}</span>
           </div>
         )}
+        {flyoutMenuItems}
```

Comment:
what is the best way to implement this? seems like it needs an event listener like `onMouseEnter` or `onMouseOver`...